### PR TITLE
add option save_database for amrfinder

### PR DESCRIPTION
**Pull request by louperelo**
closes issue #100 

## PR checklist

- [ ] This comment contains a description of changes (with reason).
- [ ] If you've fixed a bug or added code that should be tested, add tests!
  - [ ] If you've added a new tool - have you followed the pipeline conventions in the [contribution docs](https://github.com/nf-core/funcscan/tree/master/.github/CONTRIBUTING.md)
  - [ ] If necessary, also make a PR on the nf-core/funcscan _branch_ on the [nf-core/test-datasets](https://github.com/nf-core/test-datasets) repository.
- [ ] Make sure your code lints (`nf-core lint`).
- [ ] Ensure the test suite passes (`nextflow run . -profile test,docker --outdir <OUTDIR>`).
- [ ] Usage Documentation in `docs/usage.md` is updated.
- [ ] Output Documentation in `docs/output.md` is updated.
- [ ] `CHANGELOG.md` is updated.
- [ ] `README.md` is updated (including new tool citations and authors/contributors).

### CODE REVIEW

**Review by jasmezz — APPROVED** [review]
Thanks! Maybe we can mention all tools that are affected by this parameter in the `nextflow_schema.json`. Either you do it here or we do it later in the parameter clean-up.

**Comment by louperelo** [comment]
Did you mean like this?

File: nextflow_schema.json [hunk 1]
```diff
@@ -92,7 +92,7 @@
                 "save_databases": {
                     "type": "boolean",
                     "fa_icon": "fas fa-save",
-                    "description": "Specify whether to save pipeline-downloaded databases in the --outdir",
+                    "description": "Specify whether to save pipeline-downloaded databases in <outdir>/databases. This applies to databases used by deepARG, AMRfinderPlus and antiSMASH",
```
Comment (jasmezz) [inline nextflow_schema.json:95]:
```suggestion
                    "description": "Specify whether to save pipeline-downloaded databases in <outdir>/databases. This applies to databases used by deepARG, AMRfinderPlus and antiSMASH.",
```

Comment (jasmezz) [reply nextflow_schema.json:95]: Yesss 👍

**Pull request merged**